src/store/socket.js

```diff
@@ -3,27 +3,29 @@ import { gotMessages } from '../store/messages/actions'
 
 let token = ''
 
-const socket = io('http://localhost:5000');
+const socket = io('http://localhost:5000', {
+    query: { token: token }
```

Comment:
The same. Use the second option.
1)` query: { token: token }`
2)` query: { token }`